docs/testing.rst

```diff
@@ -265,3 +265,91 @@ does hurt to double check though.
 
 .. _Betamax: https://github.com/sigmavirus24/betamax
 .. _cassettes: https://betamax.readthedocs.org/en/latest/cassettes.html
+
+Asserting using the assert statement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
So this needs a better title and you need to ensure the number of `~`s is the length of the title.